pkg/installmanager/installmanager.go

```diff
@@ -1630,7 +1630,11 @@ func waitForProvisioningStage(m *InstallManager) error {
 	if err != nil {
 		return errors.Wrap(err, "could not get the GVK for clusterprovisions")
 	}
-	restClient, err := apiutil.RESTClientForGVK(gvk, false, config, scheme.Codecs)
+	rc, err := rest.RESTClientFor(config)
```

Comment:
I remember we talked about this, but I'm not sure why we ended up with RESTClientFor(), and then dereferencing the client from the result. We should just use HTTPClientFor().